scripts/build-dist.sh

```diff
@@ -16,25 +17,33 @@ case "$(uname -s)" in
     ;;
 esac
 
+version=`node -e "console.log(JSON.parse(fs.readFileSync('package.json')).version)"`
```

Comment:
We could use `jq` instead of this but we are guaranteed to have `node` so why not take advantage of that.